include/robot_interfaces/sensor_data.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <iostream>
+#include <memory>
+#include <string>
+
+#include <time_series/time_series.hpp>
+
+namespace robot_interfaces
+{
+template <typename Type>
+using Timeseries = time_series::TimeSeries<Type>;
```

Comment:
Better move this inside the class to avoid collisions in the robot_interfaces namespace.